batch2/batch/driver/instance_pool.py

```diff
@@ -131,7 +131,7 @@ def add_instance(self, instance):
             'name': machine_name,
             'machineType': f'projects/{PROJECT}/zones/{ZONE}/machineTypes/n1-{WORKER_TYPE}-{WORKER_CORES}',
             'labels': {
-                'role': 'batch2-agent',
+                'role': 'batch2-worker',
```

Comment:
I looked at the firewall rules and they're still using batch2-agent. How did you get this change to work?

Reply:
Ah, I didn't realize this was used by the firewall.  I thought we were just calling things "worker" instead of "agent".  I didn't do anything.  We should understand this, but I'll back off for now.

Reply:
It's not hard to change the firewall rule. I can't remember if the tag was used elsewhere though. I'll put it on my to-do list to figure this out.

Reply:
I mostly think we should hold off on the Big Rename(tm) for when we remove batch and rename batch2 => batch.  Let's do it then.